server/src/main/java/io/crate/metadata/upgrade/IndexTemplateUpgrader.java

```diff
@@ -49,18 +53,22 @@
 import static org.elasticsearch.common.settings.AbstractScopedSettings.ARCHIVED_SETTINGS_PREFIX;
 import static org.elasticsearch.common.settings.IndexScopedSettings.DEFAULT_SCOPED_SETTINGS;
 
+import javax.annotation.Nonnull;
+
 public class IndexTemplateUpgrader implements UnaryOperator<Map<String, IndexTemplateMetadata>> {
 
     private final Logger logger;
     public static final String TEMPLATE_NAME = "crate_defaults";
+    private static final String BROKEN_PREFIX = "broken_";
```

Comment:
I'd would use a suffix instead, this way possible ordering of table names would remain the same. But maybe my personal preference..